simulator/src/i18n.js

```
@@ -1,17 +1,23 @@
 import Banana from 'banana-i18n';
 
 const banana = new Banana();
-banana.setLocale(window.locale);
-const { locale } = banana;
-const finalFallback = 'en';
-// object with default language preloaded
+banana.setLocale("ar");
```

ESLint

[error] Strings must use singlequote.

(quotes)